app/views/devise/passwords/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<% content_for :title do "Forgot Password" end %>
+
+<% content_for :alerts do render "application/alerts" end %>
+
+<%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post, class: "form-signin" }) do |f| %>
+  <h2 class="form-signin-heading">Forgot your password?</h2>
+  <p class="centered-text">Send me reset password instructions</p>
+
+  <%= yield :alerts %>
```

Comment:
But you have a `content_for :alerts` a few lines above in this file. Is it only when it's nested that it causes problems?